doc/en/components/dock-manager.md

```diff
@@ -325,6 +325,47 @@ The Dock Manager component raises events when specific end-user interactions are
 this.dockManager.addEventListener('paneClose', ev => console.log(ev.detail));
 ```
 
+## Keyboard Navigation
+
+Keyboard navigation in the **Dock Manager** provides a rich variety of keyboard interactions for the user. It enhances the accessibility of the **Dock Manager** and allows to navigate through - documents, tabs and panes. With keyboard navigation the user can traverse over in tabs, which are inside in a [Document host]({environment:infragisticsBaseUrl}/products/ignite-ui/dock-manager/docs/typescript/latest/interfaces/igcdocumenthost.html), also there is oportunity for spliting the view in multiple directions when there is a [Tab group]({environment:infragisticsBaseUrl}/products/ignite-ui/dock-manager/docs/typescript/latest/interfaces/igctabgrouppane.html) with multiple tabs in it. Another key combinations is for docking different type of panes to the [`rootPane`]({environment:infragisticsBaseUrl}/products/ignite-ui/dock-manager/docs/typescript/latest/interfaces/igcdockmanagerlayout.html#rootpane). 
+
+
+Here is a full set of the key combinations:
+
+### Docking Combinations
```

Comment:
Split sections to be as following - Docking, Navigating and Other (for close). Also add 'ALT + F6' and 'ALT + SHIFT + F6' combinations.